GOVERNANCE.md

```diff
@@ -0,0 +1,5 @@
+# Node.js GitHub Bot Project Governance
+
+The Node.js GitHub Bot is overseen by the Node.js Technical Steering Committee.
+https://github.com/nodejs/TSC
+
```

Comment:
Everything in the GitHub org is under the TSC. The TSC, delegates many/most things to a dedicated group/WG/Committee.

If someday we want to create a new WG that manages the bot, we could request the TSC assigns the responsibility to the WG. But for now, it is delegated to the members of the @nodejs/github-bot team.